keyboards/murcielago/rev1/config.h

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 elagil
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x0000
```

Comment:
I'd recommend a unique (inside the QMK directory) VID.
You can use the hex value of your initials or something.

Reply:
You can use https://www.the-sz.com/products/usbid/index.php to verify if the VID is assigned already

Reply:
Ok, I chose a unique ID that is also not assigned in the usbid index.